x-pack/plugins/observability_solution/entity_manager/server/lib/entities/delete_transforms.ts

```diff
@@ -0,0 +1,79 @@
+/*
```

Comment:
Can we export a single deleteTransforms like the startTransform function and use it where needed ?